src/Pages/Dashboard/Dashboard.jsx

```diff
@@ -40,6 +40,10 @@ const Dashboard = (props) => {
     navigate("/license/generate");
   };
 
+  const onSearchProductKey = () => {
+    navigate("/license/search");
```

Comment:
Create and export constant values for route path in routes.js file.

For e.g.,
export const LICENSE_SEARCH_PAGE_ROUTE = "/license/search";

use it like: 
navigate(LICENSE_SEARCH_PAGE_ROUTE);